rubble/Cargo.toml

```diff
@@ -14,10 +14,13 @@ edition = "2018"
 byteorder = { version = "1.3.1", default-features = false }
 bitflags = "1.0.4"
 uuid = { version = "0.7.4", default-features = false }
-bbqueue = "0.3.2"
+bbqueue = { git = "https://github.com/chocol4te/bbqueue", features = ["cortex-m"] }
 
 # If the `log` feature is enabled, the `log` crate's macros will be called at various points to dump
 # packets, state, and events. By default, it is disabled.
 [dependencies.log]
-version = "0.4.6"
+git = "https://github.com/rust-lang-nursery/log"
 optional = true
+
+[features]
+cortex-m = ["bbqueue/cortex-m"]
```

Comment:
Please add documentation to this features (in particular, mention that it must be enabled if Rubble is used on thumbv6 targets). Perhaps it's better to rename the feature to `thumbv6` for clarity?

Reply:
Fixed (locally)